packages/amplify-graphql-schema-test-library/src/index.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-bitwise */
+export const enum TransformerVersion {
+  v1 = 1 << 0,
+  v2 = 1 << 1,
+  all = ~0,
+}
+
+export const enum TransformerPlatform {
+  none = 0,
+  api = 1 << 0,
+  dataStore = 1 << 1,
+  js = 1 << 2,
+  jsDataStore = 1 << 3,
+  ios = 1 << 4,
+  iosDataStore = 1 << 5,
+  android = 1 << 6,
+  androidDataStore = 1 << 7,
+  flutter = 1 << 8,
+  flutterDataStore = 1 << 9,
+  all = ~0,
+}
+
+export type TransformerSchema = {
+  description: string;
+  transformerVersion: TransformerVersion;
+  supportedPlatforms: TransformerPlatform;
+  sdl: string;
+};
+
+export const schemas: { [key: string]: TransformerSchema } = {
+  '@model-simple': {
```

Comment:
I wonder if this becomes a bit more readable/discoverable if we separate test schemas into separate files, e.g. https://github.com/aws-amplify/amplify-codegen-ui/blob/main/packages/test-generator/lib/components/componentWithActionSignOut.json and https://github.com/aws-amplify/amplify-codegen-ui/blob/main/packages/test-generator/lib/components/index.ts

Reply:
More of a nitpick though, that's easy enough to separate later if we deem it necessary.